src/plugins/intel_gna/src/transformations/remove_pre_post_processing.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transformations/remove_pre_post_processing.hpp"
+
+#include <openvino/cc/ngraph/itt.hpp>
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/manager.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
+
+#include "openvino/pass/pass.hpp"
+
+using namespace ov::opset9;
+using namespace ov::intel_gna::pass;
+
+namespace {
+
+ov::Shape SqueezeShape(const ov::Shape& shape) {
+    ov::Shape squeezed_shape;
+    std::copy_if(shape.begin(), shape.end(), std::back_inserter(squeezed_shape), [](size_t x) {
+        return x != 1;
+    });
+    return squeezed_shape;
+}
+
+bool IsPreprocessingLayerSuppported(std::shared_ptr<ngraph::Node>& layer) {
```

Comment:
Does this method check if Preprocessing is supported or required?

Reply:
That it is required, will fix the name

Reply:
changed the name